components/blocks/SiteFooter/SiteFooter.test.ts

```diff
@@ -11,6 +11,6 @@ describe('<SiteFooter />', () => {
   it('renders correctly with the copyright info', () => {
     const { getByText } = stubbedRender(SiteFooter)
 
-    expect(getByText('Copyright 2021')).toBeInTheDocument()
+    expect(getByText('Copyright 2022')).toBeInTheDocument()
```

Comment:
We could add a build step that automatically updates this. Like a `sed` or `rg` call or something

Reply:
Went ahead and just had it pull the current year from a new date object instead

Reply:
That definitely works too ha